### ISSUE TITLE
Misleading message about missing dependency

### ISSUE THREAD

**Issue by Jlo6CTEP** [RaccoonlabDev/innopolis_vtol_dynamics#27]
When I have first launched autopilot_configure.py, I was greeted with the following error
```
kotik@YURIY-DESKTOP:~/innopolis_vtol_dynamics$ ./scripts/autopilot_configure.py
Error: No module named 'mavlink_tools'
Try: `python3 -m pip install --index-url https://test.pypi.org/simple/ mavlink_tools`.
```
I then installed the package, and launched the autopilot_configure.py once again, and got this message
```
kotik@YURIY-DESKTOP:~/innopolis_vtol_dynamics$ ./scripts/autopilot_configure.py
Error: No module named 'pymavlink'
Try: `python3 -m pip install --index-url https://test.pypi.org/simple/ mavlink_tools`.
```

It looks like it suggests me to install `mavlink_tools` once again (maybe `pymavlink` is a part of `mavlink_tools` and this is what the "Error: No module" means), however, it is not the case. `pymavlink` should be installed separately via

```
pip install pymavlink
```

and then autopilot_configure.py works

My suggestion is to either remove the line with "Try python3 ..... " entirely and let user manage their dependencies, or supply requirements.txt file, or add a separate prompt with instructions for pymavlink and mavlink_tools respectively, because for now this message misleads user into thinking that they are missing mavlink_tools package, not pymavlink.

**Comment by PonomarevDA** [comment]
You got it right. I fixed it here: #28.

**Comment by Jlo6CTEP** [comment]
Great!
